src/functions.rs

```diff
@@ -676,27 +690,30 @@ pub fn first_value(
 }
 
 // nth_value requires a non-expr argument
-#[pyfunction]
-pub fn nth_value(
-    expr: PyExpr,
-    n: i64,
-    distinct: Option<bool>,
-    filter: Option<PyExpr>,
-    order_by: Option<Vec<PyExpr>>,
-    null_treatment: Option<NullTreatment>,
-) -> PyResult<PyExpr> {
-    let agg_fn = datafusion::functions_aggregate::nth_value::nth_value(vec![expr.expr, lit(n)]);
-    add_builder_fns_to_aggregate(agg_fn, distinct, filter, order_by, null_treatment)
-}
+// #[pyfunction]
+// #[pyo3(signature = (expr, n, distinct=None, filter=None, order_by=None, null_treatment=None))]
+// pub fn nth_value(
+//     expr: PyExpr,
+//     n: i64,
+//     distinct: Option<bool>,
+//     filter: Option<PyExpr>,
+//     order_by: Option<Vec<PySortExpr>>,
+//     null_treatment: Option<NullTreatment>,
+// ) -> PyResult<PyExpr> {
+//     // @todo: Commenting this function out for now as it requires some reworking
```

Comment:
Another area that needs some love. It looks like the `nth_value()` function had a signature change

Reply:
I fixed up `nth_value` in my commit